### PR TITLE
luminous: cls_rgw test is only run in rados suite: add it to rgw suite as well

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/25025

### CODE REVIEW

**Comment by smithfarm** [comment]
Waiting for https://github.com/ceph/ceph/pull/24034 to be merged, then rebase.

**Comment by smithfarm** [comment]
@cbodley This makes the contents of the `qa/suites/rgw/verify/tasks/` directory in luminous resemble the same in master, except the branch names are changed from `ceph-master` to `ceph-luminous`.

Look right to you?

**Comment by yuriw** [comment]
[wip-yuri3-testing-2018-09-17-1952-luminous](https://trello.com/c/tDY6qyzl/407-wip-yuri3-testing-2018-09-17-1952-luminous)

File: qa/suites/rgw/verify/tasks/ragweed.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tasks:
+- ragweed:
+    client.0:
+      force-branch: ceph-luminous
+      rgw_server: client.0
+      stages: prepare,check
```
Comment (cbodley) [inline qa/suites/rgw/verify/tasks/ragweed.yaml]: oops - the ragweed task doesn't appear to exist in luminous. can you please remove this ragweed.yaml file?

**Comment by yuriw** [comment]
[wip-yuri4-testing-2018-09-28-1911-luminous](https://trello.com/c/59wiOnS7/421-wip-yuri4-testing-2018-09-28-1911-luminous)

**Pull request merged**